pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -57,6 +61,18 @@ def pytest_runtest_teardown(item):
         item.config.hook.pytest_check_for_leftover_pulp_objects(config=item.config)
 
 
+def pytest_collection_modifyitems(config, items):
+    # Skip nightly tests by default
+    # https://docs.pytest.org/en/7.1.x/example/simple.html#control-skipping-of-tests-according-to-command-line-option
+    if config.getoption("--nightly"):
```

Comment:
```suggestion
    if not config.getoption("--nightly"):
```

Consider having the `not` here make this branch do things instead of the `return` without the `not`.

Reply:
I took the example from here: https://docs.pytest.org/en/6.2.x/example/simple.html#control-skipping-of-tests-according-to-command-line-option